README.md

```diff
@@ -76,3 +54,8 @@ so. This is to prevent scenarios where dev code sends real funds to unrecoverabl
 - `<chainId>[n].apiName` (optional): The name of the API provider.
 - `<chainId>[n].providerXpub`: The extended public key of the sponsor address.
 - `<chainId>[n].sponsor`: The sponsor address to derive the destination wallet.
+- `<chainId>[n].lowBalance.value`: The wallet balance value below which an alert is triggered.
+- `<chainId>[n].lowBalance.unit`: The token units used to parse the `lowBalance.value` for balance check (i.e. ethers,
+  wei, etc).
+- `<chainId>[n].lowBalance.criticalPercentage`: The percentage below the `lowBalance.value` used to trigger a critical
+  alert.
```

Comment:
Item #2 on [this issue](https://github.com/api3dao/operations-database/issues/285) talks about a % but I'm not sure if using a number would not make it easier to configure. For example, instead of:
```
{
  value: 0.5
  unit: 'ether',
  criticalPercentage: 60
}
```
we could use this:
```
{
  value: 0.5
  unit: 'ether',
  criticalValue: 0.2
}
```

Reply:
Yeah I think this is a good approach, so feel free to make the update if you'd prefer that as well 😄